### ISSUE TITLE
no camera firmware/ where can i get firmwares from? 

### ISSUE THREAD

**Issue by ShinyRobotVR** [wladimir-computin/FimiX8-SDFG#12]
Hi could you help after killing my battery. i messed up the irmware it seems just have new battery but no camera fimewar detected. can i ask where i can get the firmwares from please

**Comment by J3r3m38** [comment]
I have exactly the same problem......

**Comment by ShinyRobotVR** [comment]
I got firmware from fimi support that fixed it

**Comment by J3r3m38** [comment]
can you send it to me by a google drive link? and the precedence you used? thanks

> I got firmware from fimi support that fixed it
> […](#)
> On 29 Mar 2020, at 21:45, J3r3m38 ***@***.***> wrote: I have exactly the same problem...... — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe.



> I got firmware from fimi support that fixed it
> […](#)
> On 29 Mar 2020, at 21:45, J3r3m38 ***@***.***> wrote: I have exactly the same problem...... — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by LACCMAN** [comment]
Hy!  That's the problem.  Does not update fc, no image.  Can you help me send me firmware?  Fimi support did not help me ... 😕 Thank you!

**Comment by LACCMAN** [comment]
> A fimi támogatástól kaptam firmware-t, amely javította
> [...](#)
> On 29 Mar 2020, at 21:45, J3r3m38 ***@***.***> wrote: I have exactly the same problem...... — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe.

Hy!  That's the problem.  Does not update fc, no image.  Can you help me send me firmware (Google drive lik...)?  Fimi support did not help me ... 😕
For me the generator works but it doesn't fix the problem ..
Thank you!

**Comment by shadowoh** [comment]
anyone that got this problem feel free to ask me for help.
i had same issue and fixed it by myself.

my mail: tpurta8@gmail.com

good bye ;)

**Comment by budallas** [comment]
> I got firmware from fimi support that fixed it
> […](#)
> On 29 Mar 2020, at 21:45, J3r3m38 ***@***.***> wrote: I have exactly the same problem...... — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe.

Could you share us the upgrade procedure?

**Comment by shadowoh** [comment]
---------- Forwarded message ---------
Od: Thomas Purta <tpurta8@gmail.com>
Date: sob., 6 cze 2020, 23:22
Subject: Re: Fimi X8 se missing camera and vision firmware.
To: J3R3M <o0oomiimiioo0o13@gmail.com>


HI.
So You got same problem as i had... :)
Download files from my link

https://drive.google.com/file/d/17cJhcfhgwZHTSCoYMesqO1hlg3z-_CZU/view?usp=sharing

delete that Ambarella EVK download - I bet u tried to recover camera using
that software... ;) wont help You anyway :)

install everything from my package, except .efk file ofc (camera firmware)
:)

Put drone into "hard download mode" - press and hold "button 1" and swipe
2nd button 4 times left and right. -> release buttons -> rear light
blinking, front red and green are OFF - means drone is put into "hard
download mode"
Now open DirectUSB program. Connect Fimi x8se using micro USB cable.
In DirectUSB set "Board: A12evk" and Firmware programming set that .efk
file I sent You or maybe You already got it.(camera firmware). Mark "VERIFY"
Press CONNECT.
PRESS DOWNLOAD

wait 2-3 mins.

If successful, Drone camera is unblocked :)

Worked for mine Fimi x8se

GOOD LUCK !!!



sob., 6 cze 2020 o 22:50 J3R3M <o0oomiimiioo0o13@gmail.com> napisał(a):

>
> Hi, so I come to see you for help on my fimi X8 se .... or I no longer
> have video feedback and the frimware camera and vision is on n / a. I have
> several times reinstall the frimware but nothing helps ..... thanks in
> advance for help. Best regards, J3r3m38
>
>
> <https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail> Garanti
> sans virus. www.avast.com
> <https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
> <#m_5572837228590657818_m_-8276470028203008586_DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>
>

**Comment by shadowoh** [comment]
---------- Forwarded message ---------
Od: Thomas Purta <tpurta8@gmail.com>
Date: sob., 6 cze 2020, 23:25
Subject: Re: Fimi X8 se missing camera and vision firmware.
To: J3R3M <o0oomiimiioo0o13@gmail.com>


and plz remember to run DirectUSB as administrator

sob., 6 cze 2020 o 23:22 Thomas Purta <tpurta8@gmail.com> napisał(a):

> HI.
> So You got same problem as i had... :)
> Download files from my link
>
>
> https://drive.google.com/file/d/17cJhcfhgwZHTSCoYMesqO1hlg3z-_CZU/view?usp=sharing
>
> delete that Ambarella EVK download - I bet u tried to recover camera using
> that software... ;) wont help You anyway :)
>
> install everything from my package, except .efk file ofc (camera firmware)
> :)
>
> Put drone into "hard download mode" - press and hold "button 1" and swipe
> 2nd button 4 times left and right. -> release buttons -> rear light
> blinking, front red and green are OFF - means drone is put into "hard
> download mode"
> Now open DirectUSB program. Connect Fimi x8se using micro USB cable.
> In DirectUSB set "Board: A12evk" and Firmware programming set that .efk
> file I sent You or maybe You already got it.(camera firmware). Mark "VERIFY"
> Press CONNECT.
> PRESS DOWNLOAD
>
> wait 2-3 mins.
>
> If successful, Drone camera is unblocked :)
>
> Worked for mine Fimi x8se
>
> GOOD LUCK !!!
>
>
>
> sob., 6 cze 2020 o 22:50 J3R3M <o0oomiimiioo0o13@gmail.com> napisał(a):
>
>>
>> Hi, so I come to see you for help on my fimi X8 se .... or I no longer
>> have video feedback and the frimware camera and vision is on n / a. I have
>> several times reinstall the frimware but nothing helps ..... thanks in
>> advance for help. Best regards, J3r3m38
>>
>>
>> <https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail> Garanti
>> sans virus. www.avast.com
>> <https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
>> <#m_-8410840004032165756_m_1782095346221284215_m_-8276470028203008586_DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>
>>
>

**Comment by shadowoh** [comment]
---------- Forwarded message ---------
Od: Thomas Purta <tpurta8@gmail.com>
Date: niedz., 7 cze 2020, 00:28
Subject: Re: Fimi X8 se missing camera and vision firmware.
To: J3R3M <o0oomiimiioo0o13@gmail.com>


That two buttons near port and sd card slot

Button 1 is smaller
Button 2 is slider.u can move it right and left

niedz., 7 cze 2020, 00:26 użytkownik J3R3M <o0oomiimiioo0o13@gmail.com>
napisał:

> Bouton 1 ? and bouton 2 ?
>
>
> <https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail> Garanti
> sans virus. www.avast.com
> <https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
> <#m_-8906235972065438885_m_5774437751434374327_DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>
>

**Comment by shadowoh** [comment]
---------- Forwarded message ---------
Od: Thomas Purta <tpurta8@gmail.com>
Date: niedz., 7 cze 2020, 00:30
Subject: Re: Fimi X8 se missing camera and vision firmware.
To: J3R3M <o0oomiimiioo0o13@gmail.com>


To enter this mode turn on drone. Wait 20-30s and then press and hold
button 1 (smaller) and slide button 2 four times left and right (while
keeping button 1 pressed!!!)
Then release.both

niedz., 7 cze 2020, 00:28 użytkownik Thomas Purta <tpurta8@gmail.com>
napisał:

> That two buttons near port and sd card slot
>
> Button 1 is smaller
> Button 2 is slider.u can move it right and left
>
> niedz., 7 cze 2020, 00:26 użytkownik J3R3M <o0oomiimiioo0o13@gmail.com>
> napisał:
>
>> Bouton 1 ? and bouton 2 ?
>>
>>
>> <https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail> Garanti
>> sans virus. www.avast.com
>> <https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
>> <#m_-4860301011191761667_m_-8906235972065438885_m_5774437751434374327_DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>
>>

**Comment by Tomiz-opz** [comment]
Do you have idea where to get efk file for X8 2020 ? Force/hard download doesn't work similarly with 2020 version, I did not manage to get 2020 to mode red/green lights are of and backlight is blinking. Using 1 button + 2 gets X8 2020 to turn on red/green lights and switch off backlight (hard download mode indicator in 2020 ?).   DirectUsb doesn't recognize X8 2020...

**Comment by shadowoh** [comment]
Do You have any experience with 2018 and hard download mode?

You need to keep small button pressed while sliding second button

I dont have efk file... Ask fimi support

wt., 23 cze 2020, 08:57 użytkownik Tomiz-opz <notifications@github.com>
napisał:

**Comment by Tomiz-opz** [comment]
Yes I know how to turn hard download on X8 2018. Do you have contacts in Fimi ? Their support is giving same copy/pasted answers while asking help -> neverending loop.

**Comment by shadowoh** [comment]
support@fimi.com

Only this.... This is just fail company....

wt., 23 cze 2020, 09:09 użytkownik Tomiz-opz <notifications@github.com>
napisał:

**Comment by Tomiz-opz** [comment]
Yes, really failed company. Their support is nothing. I'm planning to dispute this X8 in credit card company -> I have no time to talk with copy paste support.

**Comment by Tomiz-opz** [comment]
This is everything I am getting from Fimi support. They don't mind about context of emails. They just copy paste the same message .   I asked about efk files and I got this:

"
2020-06-23 15:56

Dear customer,Please try to update the camera firmware to the latest version by using TF card. You can download the latest camera firmware refer to this link: https://www.fimi.com/support/download/view?id=21Please format the TF card first when you start to update the camera firmware.

Best regards,
--
Karry
"

**Comment by budallas** [comment]
> This is everything I am getting from Fimi support. They don't mind about context of emails. They just copy paste the same message . I asked about efk files and I got this:
> 
> "
> 2020-06-23 15:56
> 
> Dear customer,Please try to update the camera firmware to the latest version by using TF card. You can download the latest camera firmware refer to this link: https://www.fimi.com/support/download/view?id=21Please format the TF card first when you start to update the camera firmware.
> 
> ## Best regards,
> Karry
> "

Fimi supprt is very disappointing .I  have to waiting days for their stupid answers.I`m having the same problem with the new received drone.I will return back. with the 
![IMG_20200613_174535ew](https://user-images.githubusercontent.com/67186114/85412193-03085b80-b572-11ea-9992-b77f219c9a06.jpg)
hope I can get my money back.

**Comment by Tomiz-opz** [comment]
I want only get new X8. It is good drone but there are broken ones also. Customer service should be improved a lot. Next drones I will consider customer service quality, not only for item.

**Comment by jony4t** [comment]
> 
> ---------- Forwarded message --------- Od: Thomas Purta <tpurta8@gmail.com> Date: sob., 6 cze 2020, 23:22 Subject: Re: Fimi X8 se missing camera and vision firmware. To: J3R3M <o0oomiimiioo0o13@gmail.com> HI. So You got same problem as i had... :) Download files from my link https://drive.google.com/file/d/17cJhcfhgwZHTSCoYMesqO1hlg3z-_CZU/view?usp=sharing delete that Ambarella EVK download - I bet u tried to recover camera using that software... ;) wont help You anyway :) install everything from my package, except .efk file ofc (camera firmware) :) Put drone into "hard download mode" - press and hold "button 1" and swipe 2nd button 4 times left and right. -> release buttons -> rear light blinking, front red and green are OFF - means drone is put into "hard download mode" Now open DirectUSB program. Connect Fimi x8se using micro USB cable. In DirectUSB set "Board: A12evk" and Firmware programming set that .efk file I sent You or maybe You already got it.(camera firmware). Mark "VERIFY" Press CONNECT. PRESS DOWNLOAD wait 2-3 mins. If successful, Drone camera is unblocked :) Worked for mine Fimi x8se GOOD LUCK !!! sob., 6 cze 2020 o 22:50 J3R3M <o0oomiimiioo0o13@gmail.com> napisał(a):
> […](#)
> Hi, so I come to see you for help on my fimi X8 se .... or I no longer have video feedback and the frimware camera and vision is on n / a. I have several times reinstall the frimware but nothing helps ..... thanks in advance for help. Best regards, J3r3m38 <https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail> Garanti sans virus. [www.avast.com](http://www.avast.com) <https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail> <#m_5572837228590657818_m_-8276470028203008586_DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

Any video of how the drone is put in download mode?

**Comment by vietdund** [comment]
![image](https://user-images.githubusercontent.com/79757832/111071833-8dcb2500-850a-11eb-9cea-c5d5a99964d7.png)
i have tried but cannot conect

**Comment by VorasLT** [comment]
> ---------- Forwarded message --------- Od: Thomas Purta <tpurta8@gmail.com> Date: sob., 6 cze 2020, 23:22 Subject: Re: Fimi X8 se missing camera and vision firmware. To: J3R3M <o0oomiimiioo0o13@gmail.com> HI. So You got same problem as i had... :) Download files from my link https://drive.google.com/file/d/17cJhcfhgwZHTSCoYMesqO1hlg3z-_CZU/view?usp=sharing delete that Ambarella EVK download - I bet u tried to recover camera using that software... ;) wont help You anyway :) install everything from my package, except .efk file ofc (camera firmware) :) Put drone into "hard download mode" - press and hold "button 1" and swipe 2nd button 4 times left and right. -> release buttons -> rear light blinking, front red and green are OFF - means drone is put into "hard download mode" Now open DirectUSB program. Connect Fimi x8se using micro USB cable. In DirectUSB set "Board: A12evk" and Firmware programming set that .efk file I sent You or maybe You already got it.(camera firmware). Mark "VERIFY" Press CONNECT. PRESS DOWNLOAD wait 2-3 mins. If successful, Drone camera is unblocked :) Worked for mine Fimi x8se GOOD LUCK !!! sob., 6 cze 2020 o 22:50 J3R3M <o0oomiimiioo0o13@gmail.com> napisał(a):
> […](#)
> Hi, so I come to see you for help on my fimi X8 se .... or I no longer have video feedback and the frimware camera and vision is on n / a. I have several times reinstall the frimware but nothing helps ..... thanks in advance for help. Best regards, J3r3m38 <https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail> Garanti sans virus. [www.avast.com](http://www.avast.com) <https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail> <#m_5572837228590657818_m_-8276470028203008586_DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

Thanks! This one helped me after unsuccessful try to update camera firmware via sd-card.

**Comment by shadowoh** [comment]
:))))

pon., 18 wrz 2023, 09:37 użytkownik VorasLT ***@***.***>
napisał:

**Comment by Chellux** [comment]
hi

i tried it many times, the software says "ok" but i dont get any "camera" or "vision" firmware showed.. just N/A

its a 2019 x8

it sucks because i get that shitty "VPU ERROR" :/
